lib/aur-sync

```diff
@@ -231,7 +232,7 @@ mkdir -p "$AURDEST"
 cd_safe "$tmp"
 
 # retrieve path to local repo (#448)
-aur repo "${repo_args[@]}" --list --status-file=db >db_info
+aur repo --pacman-conf "${pacman_conf:-/etc/pacman.conf}" "${repo_args[@]}" --list --status-file=db >db_info
```

Comment:
Would be simpler to add `repo_args+=(--pacman-conf "$1")` to L161 instead of `pacman_conf="$1"`